### PR TITLE
aws vpc peering config

### PR DESCRIPTION
**Pull request by NissesSenap**

### CODE REVIEW

File: modules/aws/core/vpc.tf [hunk 1]
```diff
@@ -166,29 +198,73 @@ resource "aws_route_table_association" "private" {
 # VPC Peering
 resource "aws_vpc_peering_connection" "this" {
```
Comment (NissesSenap) [inline modules/aws/core/vpc.tf:199]: Instead of this should I call it requester?

File: modules/aws/core/variables.tf [hunk 2]
```diff
@@ -31,22 +38,30 @@ variable "vpc_config" {
   })
 }
 
-variable "vpc_peering_enabled" {
-  description = "If true vpc peering will be configured"
-  type        = bool
-  default     = true
+variable "vpc_peering_config_requester" {
+  description = "VPC Peering configuration"
```
Comment (phillebaba) [inline modules/aws/core/variables.tf:42]: Update the description.

File: modules/aws/core/variables.tf [hunk 3]
```diff
@@ -31,22 +38,30 @@ variable "vpc_config" {
...
+    peer_owner_id          = string
+    peer_vpc_id            = string
+    destination_cidr_block = string
+  }))
+  default = []
 }
 
-variable "vpc_peering_config" {
+variable "vpc_peering_config_accepter" {
   description = "VPC Peering configuration"
```
Comment (phillebaba) [inline modules/aws/core/variables.tf:53]: Ditto

File: modules/aws/core/variables.tf [hunk 4]
```diff
@@ -31,22 +38,30 @@ variable "vpc_config" {
...
-    destination_cidr_block = ""
-    peer_owner_id          = ""
-    peer_vpc_id            = ""
-  }
+  }))
+  default = []
+}
+
+
+variable "internet_gateway_enabled" {
```
Comment (phillebaba) [inline modules/aws/core/variables.tf:63]: Do we need this if we can just check the count of the public subnets?

**Review by phillebaba — APPROVED** [review]

**Review by phillebaba — APPROVED** [review]
LGTM

**Pull request merged**